### DISCUSSION TITLE
How to install qurato in windows server as a normal user not admin

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [quarto-dev/quarto-cli#11204] [Q&A]
### Description

I am trying to install nbdev on windows server with no admin privilege, the quarto-win.msi need amdin to execute and the quarto-win.zip didn't need admin but when i extract it, it gives me 2 folder one called bin and other is share. 
in the bin folder there the quarto.exe and quarto.cmd when i open quarto.exe it's just blink

**Comment by cderv** [comment]
> the quarto-win.zip didn't need admin

You want the zip file. Just download it and extract it. 

> it gives me 2 folder one called bin and other is share.

The executable is in `bin/` - you need to use the `quarto.exe` file. You can add it to PATH if you want to call it from anywhere e.g. `quarto render`

> when i open quarto.exe it's just blink

What are you expecting ? Quarto is not a GUI tool. It is a command line tool. So open a terminal and call 
`quarto --help` or `quarto --version` from this bin folder, or using the full path to the binary. 

To render a document you want to call `quarto render`

> the quarto-win.msi need admin to execute 

If you really need MSI, I believe there is some flag you can use to opt-out options that requires admin priviledge. Though the zip archive is usually enough in that case. 

Hope it helps
